### PR TITLE
refs #10244 - adding fedora support for katello-repos

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: rel-eng/tito.props [hunk 1]
```diff
@@ -9,3 +9,12 @@ blacklist =
...
 disttag = .el7
 blacklist =
+
+[katello-nightly-fedora20]
+disttag = .fc20
+whitelist = katello-repos
+
+[katello-nightly-fedora21]
+disttag = .fc21
+whitelist = katello-repos
```
Comment (ehelms) [inline rel-eng/tito.props]: Wouldn't we want to blacklist `katello-repos` so that only `katello-client-repos` builds?

**Comment by ehelms** [comment]
Do you mind also adding el5 support here for the katello-client-repos?

File: rel-eng/tito.props (hunk 1 above)
Comment (jlsherrill) [inline rel-eng/tito.props]: The whitelist/blacklist here is for the base package (not subpackages).  So here we say build 'katello-repos' for f21, and in the comps we only allow katello-client-repos into the actual yum repo.  Sure there is a rpm build on koji that is never used but the only way around that would be to split these two rpms up into separate source rpms.

**Comment by jlsherrill** [comment]
el5 added

File: rel-eng/tito.props (hunk 1 above)
Comment (ehelms) [inline rel-eng/tito.props]: That makes sense.

**Comment by ehelms** [comment]
ACK

File: rel-eng/tito.props [hunk 2]
```diff
@@ -2,6 +2,10 @@
 default_builder = tito.builder.Builder
 default_tagger = tito.tagger.ReleaseTagger
 
+[katello-nightyl-rhel5]
```
Comment (ehelms) [inline rel-eng/tito.props]: Doh, missed this typo :/

**Pull request merged**